hr_expense_invoice/tests/test_hr_expense_invoice.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# Copyright 2021-2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import base64
+
+from odoo import fields
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests import Form, tagged
+
+from odoo.addons.hr_expense.tests.common import TestExpenseCommon
+
+
+@tagged("post_install", "-at_install")
+class TestHrExpenseInvoice(TestExpenseCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, test_hr_expense_invoice=True))
```
I believe this will resolve the testing issue, similar to what happens in https://github.com/OCA/hr-expense/pull/307 (`hr_expense_invoice` + `hr_expense_advance_clearing` change the `_prepare_bills_vals()` method, so it is “_normal_” for the tests to be incorrect).